UnityProject/Assets/Scripts/Player/PlayerManager.cs

```diff
@@ -97,9 +100,18 @@ private void UpdateMe()
 		}
 
 
+		var Amove = GetMovementActions();
+
 		if (MovementControllable != null)
 		{
-			MovementControllable.ReceivePlayerMoveAction(GetMovementActions());
+			MovementControllable.ReceivePlayerMoveAction(Amove);
```

Comment:
```suggestion
			MovementControllable.ReceivePlayerMoveAction(movementActions);
```
the method suggest it receives a singular move action, but you are passing to it the result of ``GetMovementActions()``. Which one is lying?

Reply:
GetMovementActions , 
Since it's only a combination of wasd